### ISSUE TITLE
is the project dead?

### ISSUE THREAD

**Issue by remmeier** [srs/gradle-node-plugin#315]
maybe it would be good to settle for a new location for a fork to be able to do PRs again.

**Comment by sleicht** [comment]
I tried to contact @srs via mail. Let's see if and what he'll answer.

**Comment by deepy** [comment]
Since there doesn't seem to be anything happening here I forked it to https://github.com/node-gradle/gradle-node-plugin and started relocating the plugin, I mostly did this since I need #299 fixed but I kinda also want #316

**Comment by remmeier** [comment]
thx! I try to contribute Gradle build cache support the coming days.

**Comment by iamareebjamal** [comment]
@deepy Can you please enable Issues for the project?

**Comment by sleicht** [comment]
Sorry that I didn't get back. @srs did answer me that the project isn't dead and that he would be happy if someone would help manage it. I offered him my help. Unfortunately I didn't hear back since then.

**Comment by deepy** [comment]
@iamareebjamal issues are enabled now :-)

**Comment by deepy** [comment]
https://plugins.gradle.org/plugin/com.github.node-gradle.node - relocated plugin with a conditional fix for #299 now exists (since plugin portal does *not* handle snapshots)

**Comment by adamchenwei** [comment]
seems this can be a good replacement? https://plugins.gradle.org/plugin/com.liferay.node

**Comment by srs** [comment]
Hi. Sorry about being non-responsive. I have now started to work on this plugin again and I promise that I will start applying PR's. Also start adding other people wanting to contribute and help me manage this project.

**Comment by dlangerenken** [comment]
Great to have you back @srs !

**Comment by cdietrich** [comment]
nice. did you have a look at the changes in the fork https://github.com/node-gradle/gradle-node-plugin

**Comment by srs** [comment]
No, not yet. Would be happy to merge the code. What I am working on now will be tracked in 2.0 branch. Mostly towards cleaning up the plugin and make it more Kotlin friendly.

**Comment by srs** [comment]
Thanks @dlangerenken.

**Comment by imanushin** [comment]
@srs, looks like question is actual again: is project dead? For current moment I can resolve several issues, however I'd like to get owner's opinion before my coding.

**Comment by dawi** [comment]
First I would like to thank you @srs for all you time and effort you put into this project. :)

Secondly, I wanted to follow the previous comment, whether the project is dead now or not. :(

I don't want to criticise, because I understand that maintaining open source software is a very time consuming job and at a certain point maybe more stress then fun. :/ But now there is a fork of this project https://github.com/node-gradle/gradle-node-plugin and I am not sure which one to use in the future.

It would be very nice if @srs and @deepy can clarify what their plans with both projects are for the future.

Thank's in advance.

**Comment by deepy** [comment]
Since development has picked up again I'll start backporting (forwardporting?) to `node-gradle`.

As long as this repo is a one-man show I'll continue working on the node-gradle "community" fork. Which ironically also is a one-man show (and like https://github.com/srs/gradle-node-plugin/issues/315#issuecomment-457945124 also open to having more contributors).

**Comment by deepy** [comment]
@srs maybe we should schedule a quick call some day and discuss the futures of these repositories?

**Comment by azerbe** [comment]
Hey, are there any updates regarding this situation?

**Comment by dawi** [comment]
Yes. Don't use this project.
Use https://github.com/node-gradle/gradle-node-plugin instead, which is actively developed.

**Comment by blootsvoets** [comment]
Could a comment be added to the README and perhaps the Gradle plugin repository redirecting to https://github.com/node-gradle/gradle-node-plugin? I came here via #377 to find out.
